pkg/vsphere/disk/disk_manager.go

```diff
@@ -190,13 +196,12 @@ func (m *Manager) Create(op trace.Operation, newDiskURI *object.DatastorePath,
 			DiskType:    string(types.VirtualDiskTypeThin),
 			AdapterType: string(types.VirtualDiskAdapterTypeLsiLogic),
 		},
-
-		CapacityKb: capacityKB,
+		CapacityKb: config.CapacityInKB,
 	}
 
 	op.Infof("Creating vmdk for layer or volume %s", d.DatastoreURI)
 	err = tasks.Wait(op, func(ctx context.Context) (tasks.Task, error) {
-		return vdm.CreateVirtualDisk(ctx, d.DatastoreURI.String(), nil, spec)
+		return m.vdm.CreateVirtualDisk(ctx, d.DatastoreURI.String(), nil, spec)
```

Comment:
surprised to see us using CreateVirtualDisk - I'm unaware of a scenario when we're not going to then immediately attach it to a VM.
Can you clarify?